lib/galaxy/model/custom_types.py

```diff
@@ -217,11 +222,59 @@ def remove(self, value):
 } )
 
 
+def total_size(o, handlers={}, verbose=False):
```

Comment:
Yes it is in this case (you'd want the handler registry here to contain all object handlers), but it's made even safer since we don't actually use it at all, yet ;)  I wanted to (at least at first) leave the original function intact.  I'm going to follow up with a shortcut to speed it up, though.